render.yaml

```diff
@@ -27,11 +27,38 @@ services:
         - docs/**
         - '**/*.spec.ts'
 
+  - type: web
+    name: toolpad-demo
+    env: docker
+    branch: demo
+    dockerfilePath: ./docker/images/toolpad/Dockerfile.demo
+    dockerContext: ./
```

Comment:
Perhaps we can set the context to 
```yml
dockerContext: ./docker/images/toolpad/
```
? That should speed up the build as it doesn't have to upload the whole repository as a context. The docker file doesn't (and shouldn't) use anything from the repository

Reply:
I'll keep the path just in case?